cmd/telemetrygen/internal/common/tls_utils.go

```diff
@@ -82,7 +67,6 @@ func GetTLSCredentialsForHTTPExporter(caFile string, cAuth ClientAuth) (*tls.Con
 		if err != nil {
 			return nil, err
 		}
-		tlsCfg.ClientAuth = tls.RequireAndVerifyClientCert
```

Comment:
Note the reviewer: this change is an incidental cleanup. ClientAuth is irrelevant here, it's only relevant for TLS servers.